### PR TITLE
  [SYCL][Graph] Update doc for UR PR moving reset commands to a dedicated cmd-list

### PR DESCRIPTION
**Pull request by mfrancepillois**
Update the design doc.
Update the UR tag.

### CODE REVIEW

**Review by aelovikov-intel — APPROVED** [review]
I don't see what needs to be reviewed by the SYCL RT, probably imperfect CODEOWNERS.

**Comment by kbenzie** [comment]
> I don't see what needs to be reviewed by the SYCL RT, probably imperfect CODEOWNERS.

It's the design docs setup https://github.com/intel/llvm/blob/sycl/.github/CODEOWNERS#L33

**Review by EwanC — APPROVED** [review]

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -56,14 +56,8 @@ endif()
...
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit e2ee9a4720414e0a59fa9c911e9575ab564ac57c
-  # Merge: 7a5150cd be622e7c
-  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Sun Mar 10 18:02:50 2024 +0000
-  #     Merge pull request #1340 from Bensuo/ewan/coverity_cuda_update
-  #     [HIP][CUDA][Command-Buffer] Fix Coverity issues in HIP/CUDA command-buffer code
-  set(UNIFIED_RUNTIME_TAG e2ee9a4720414e0a59fa9c911e9575ab564ac57c)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/bensuo/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG maxime/improve-L0-cmd-buffer-enqueing)
```
Comment (kbenzie) [inline sycl/plugins/unified_runtime/CMakeLists.txt:60]:
```suggestion
  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
  # commit 418ad5354ca24a6dfbd01df803949855b7a6c3dd
  # Merge: d99d5f74 26682290
  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
  # Date:   Thu Mar 14 10:19:56 2024 +0000
  #     Merge pull request #1365 from Bensuo/maxime/improve-L0-cmd-buffer-enqueing
  #     [EXP][CMDBUF] Move event reset commands to dedicated cmd-list
  set(UNIFIED_RUNTIME_TAG 418ad5354ca24a6dfbd01df803949855b7a6c3dd)
```

**Review by kbenzie — COMMENTED** [review]
https://github.com/oneapi-src/unified-runtime/pull/1365 has been merged, please pull in the latest sycl branch changes, resolve the conflict and update the UR repo/tag as suggested.

**Review by kbenzie — APPROVED** [review]

**Comment by EwanC** [comment]
@intel/llvm-gatekeepers This is good to merge

**Pull request merged**